src/api/ums/umsModule.ts

```diff
@@ -101,6 +101,15 @@ export class UMSModule<
     );
   }
 
+  /**
+   * Signs out an user by just removing the token that belongs to the user/alias
+   *
+   * @param email The email that functions as an alias for the tokenManager
+   */
+  public signOut(email: string): void {
```

Comment:
Email might be not a good option. In sign user, we do not have email as an available parameter. I need to run getUser to get it. Maybe we can use some kind of a current user or something?

Reply:
Same here, email is not correct. It should be `alias`. 

Taking this example 

```
ums.signIn({
  email: 'Elon@tesla.com',
  password: 'secret-password',
  default: true,
  alias: 'Elon Musk',
}).subscribe(user => {
  // run request with current user token
}, error => {
  console.log(error)
});
```

Both the `email` and `alias` are set as the alias for the tokens.